runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsOverlay.java

```diff
@@ -188,13 +187,14 @@ public Dimension render(Graphics2D graphics)
 		final boolean onlyShowLoot = config.onlyShowLoot();
 		final DespawnTimerMode groundItemTimers = config.groundItemTimers();
 		final boolean outline = config.textOutline();
+		final int localDrawDistance = client.getScene().getDrawDistance() * Perspective.LOCAL_TILE_SIZE;
```

Comment:
There's one other change I would recommend for this, which would be to limit the scene distance to 50, as that's the maximum distance we draw clickboxes for items.

```suggestion
		final int localDrawDistance = Math.min(client.getScene().getDrawDistance(), 50) * Perspective.LOCAL_TILE_SIZE;
```

Reply:
@Steven-Posterick are you ok to make the change? I would do it but it would require opening a new PR.

Reply:
> @Steven-Posterick are you ok to make the change? I would do it but it would require opening a new PR.

Sure, accepted it now.